datastore/gcloud/aio/datastore/utils.py

```diff
@@ -1,29 +0,0 @@
-from typing import Any
-from typing import Dict
-
-from gcloud.aio.datastore.constants import FORMATTERS
-from gcloud.aio.datastore.constants import TypeName
-from gcloud.aio.datastore.constants import TYPES
-from gcloud.aio.datastore.constants import UNFORMATTERS
-
-
-def infer_type(value: Any) -> TypeName:
-    kind = type(value)
-
-    try:
-        return TYPES[kind]
-    except KeyError:
-        raise Exception(f'unsupported value type {kind}')
-
-
-def make_value(value: Any) -> Dict[str, Any]:
```

Comment:
So actually, `make_value()` and `parse_value()` predate the Value class. They were used to write/parse Values. Now that there is a Value class, I moved that logic there.